### PR TITLE
Add Filebeat module to the installation tasks

### PR DESCRIPTION
**Pull request by rshad**
Hi all.

This PR resolves https://github.com/wazuh/wazuh-chef/issues/54.

Kr,

Rshad

### CODE REVIEW

File: cookbooks/wazuh_filebeat/recipes/filebeat.rb [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 bash 'Import Wazuh module for filebeat' do 
   code <<-EOH
-  curl -s https://packages-dev.wazuh.com/3.x/filebeat/wazuh-filebeat-0.1.tar.gz | tar -xvz -C /usr/share/filebeat/module
+  curl -s https://s3-us-west-1.amazonaws.com/packages.wazuh.com/3.x/filebeat/wazuh-filebeat-0.1.tar.gz | tar -xvz -C /usr/share/filebeat/module
```
Comment (jm404) [inline cookbooks/wazuh_filebeat/recipes/filebeat.rb]: We can use the standard production link as stated in the documentation:

`https://packages.wazuh.com/3.x/filebeat/wazuh-filebeat-0.1.tar.gz`

Also adding a default attribute to manage `wazuh-filebeat-0.1-.tar.gz` is recommended in order to have a good maintainability

**Review by jm404 — CHANGES_REQUESTED** [review]
Hi @rshad 

Please have a look at requested changes

Regards!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+### Added
+
+- Update to Wazuh version 3.9.4_7.2.0
+- Added the choice to register an agent or not ([rshad](https://github.com/rshad)) [PR#50](https://github.com/wazuh/wazuh-chef/pull/50)
+
+- Added Filebeat's module installation to the installation tasks ([rshad](https://github.com/rshad)) [PR#55](https://github.com/wazuh/wazuh-chef/pull/55)
+
+### Changed
+
+- Upgraded wazuh_api to accept Vault, Data Bags or none of them to configure Wazuh API ([@jm404] (https://github.com/jm404)) [PR#47](https://github.com/wazuh/wazuh-chef/pull/47)
```
Comment (jm404) [inline CHANGELOG.md]: My name and link are not well-formatted, please have a look at it

image

File: README.md [hunk 3]
```diff
@@ -153,6 +153,19 @@ default['ossec']['agent_auth']['register'] = 'yes'
 
 In other case, we just assign a different value which is not `yes`.
 
+
+## In case you're using CentOS 6 with Kibana 7.2.0
```
Comment (jm404) [inline README.md]: Let's move this under de dependencies section, I think most of us don't completely read the README and some users might miss it.

**Review by jm404 — CHANGES_REQUESTED** [review]
Hi @rshad , added some comments, have a look at them please

Thanks!

File: README.md (hunk 3 above)
Comment (rshad) [reply README.md]: Hi @jm404,

Check the changes [here](https://github.com/wazuh/wazuh-chef/commit/59d4e6be6389fe57e017a6c47a66177280c3ede2)

Kind regards,

Rshad

File: CHANGELOG.md (hunk 2 above)
Comment (rshad) [reply CHANGELOG.md]: Hi @jm404,

Sorry, this is was a typo.

Here you have the [correction](https://github.com/wazuh/wazuh-chef/commit/60696b0ad99c51ecff81447a6b6dca80e3b78dcd).

Kind regards,

Rshad

File: cookbooks/wazuh_filebeat/recipes/filebeat.rb (hunk 1 above)
Comment (rshad) [reply cookbooks/wazuh_filebeat/recipes/filebeat.rb]: Hi @jm404,

You're right that would be the official link to the package.

Here you have the [changes](https://github.com/wazuh/wazuh-chef/commit/1a0a6fb2f4d069ed316f43f989d3a47e74211656).

Kind regards,

Rshad

**Review by jm404 — APPROVED** [review]
LGTM!

**Review by jm404 — APPROVED** [review]
LGTM!

**Pull request merged**